R/figures.R

```diff
@@ -0,0 +1,220 @@
+#' Get plot of fit and forecasted counts
+#'
+#' @param draws A dataframe containing the posterior draws with the data joined
+#' to it. This is `draws_df` output of the call to `wwinference()`
+#' @param count_data_eval A dataframe containing the count data we will
+#' evaluate the forecasts against.
```

Comment:
@kaitejohnson

Reply:
Yeah this is the output of the `wwinference()` function, so I thought it would be okay to not specify the column names since they will all be standardized. But lmk what you think.

Reply:
To resolve.

Reply:
Done, added the `col_name` argument and specified the need for the other column to be `date`